users/views.py

```diff
@@ -154,6 +155,15 @@ def user_create_confirm(request, token):
         user.confirm_token = ''
         user.is_active = True
         user.save()
+        user.circle_set.create(
+            name='Me',
+            is_editable=False
+        )
+        user.circle_set.create(
+            name='Public',
+            is_editable=False
+        )
```

Comment:
Whatever it is, I would like to have it discussed as soon as possible so I can fix that tomorrow at the latest